yt_dlp/extractor/drtv.py

```diff
@@ -372,3 +373,91 @@ def _real_extract(self, url):
             'formats': formats,
             'is_live': True,
         }
+
+
+class DRTVSeasonIE(InfoExtractor):
+    IE_NAME = 'drtv:season'
+    _VALID_URL = r'''(?x)
+                    https?://
+                        (?:
+                            (?:www\.)?(?:dr\.dk|dr-massive\.com)/drtv/(?:saeson)/
+                        )
+                        (?P<id>[\da-z_-]+)
+                    '''
+    _GEO_COUNTRIES = ['DK']
+    _TESTS = [{
+        'url': 'https://www.dr.dk/drtv/saeson/frank-and-kastaniegaarden_9008',
+        'info_dict': {
+            'id': 'frank-and-kastaniegaarden_9008',
+            'title': 'Frank & Kastaniegaarden',
+            'series': 'Frank & Kastaniegaarden',
+        },
+        'playlist_mincount': 8
+    }]
+
+    def _real_extract(self, url):
+        season_id = self._match_id(url)
+        data = self._download_json(f'https://production-cdn.dr-massive.com/api/page?device=web_browser&item_detail_expand=all&lang=da&max_list_prefetch=3&path=/saeson/{season_id}', season_id)
+
+        entries = [{
+            '_type': 'url_transparent',
```

Comment:
Why transparent?

Reply:
Honestly I couldn't find any documentation about how to build playlist extractors, so I relied on existing extractors as samples. The first extractor I found that I could understand used `url_transparent`, but I'll happily switch to `url`.

Reply:
url_transparent means the metadata returned here will override whatever is set by the the video extractor. Use `url` if all the data is also extracted by the video extractor. Otherwise, transparent is better